packages/editor-ui/src/types/assistant.types.ts

```diff
@@ -76,6 +77,7 @@ export namespace ChatRequest {
 		role: 'assistant';
 		type: 'message';
 		text: string;
+		step?: 'n8n_documentation' | 'n8n_forum';
```

Comment:
what if we add a new tool in the future? we should ensure this can be backwards compatible

Reply:
Agree with this also, will update so each step message is served from the service